petab/v2/C.py

```diff
@@ -193,35 +187,50 @@
 
 # NOISE MODELS
 
-#: Uniform distribution
-UNIFORM = "uniform"
-#: Uniform distribution on the parameter scale
-PARAMETER_SCALE_UNIFORM = "parameterScaleUniform"
-#: Normal distribution
-NORMAL = "normal"
-#: Normal distribution on the parameter scale
-PARAMETER_SCALE_NORMAL = "parameterScaleNormal"
+
+#: Cauchy distribution.
+CAUCHY = "cauchy"
+#: Chi-squared distribution.
+# TODO: "chisquare" in PEtab and sbml-distrib, but usually "chi-squared"
+CHI_SQUARED = "chisquare"
+#: Exponential distribution.
+EXPONENTIAL = "exponential"
+#: Gamma distribution.
+GAMMA = "gamma"
 #: Laplace distribution
 LAPLACE = "laplace"
-#: Laplace distribution on the parameter scale
-PARAMETER_SCALE_LAPLACE = "parameterScaleLaplace"
-#: Log-normal distribution
-LOG_NORMAL = "logNormal"
+#: Log10-normal distribution.
+LOG10_NORMAL = "log10-normal"
 #: Log-Laplace distribution
-LOG_LAPLACE = "logLaplace"
+LOG_LAPLACE = "log-laplace"
+#: Log-normal distribution
+LOG_NORMAL = "log-normal"
+#: Log-uniform distribution.
+LOG_UNIFORM = "log-uniform"
```

Comment:
Not really in-scope for this PR but is log10-uniform missing? I guess this is how we would convert `parameterScale=log10` into PEtab v2, or is that just no longer possible?

Reply:
It's not included in https://github.com/PEtab-dev/PEtab/pull/595. So, so far, it's not available in PEtab v2. Right, I guess we'd need it to upconvert v1 priors. Also log10-laplace. Best to be discussed at https://github.com/PEtab-dev/PEtab.